content/en/logs/guide/detect-unparsed-logs.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Monitor and query for unparsed logs
+kind: guide
+aliases:
+  - /logs/guide/detect-unparsed-logs/
+further_reading:
+  - link: /logs/explorer/
+    tag: Documentation
+    text: "See how to explore your logs"
+  - link: /logs/faq/log-parsing-best-practice
+    tag: Documentation
+    text: "Log Parsing - Best Practices"
+beta: true
+---
+
+<div class="alert alert-warning">
+The Unparsed logs flagging feature is currently in private beta. For more information, contact <a href="https://docs.datadoghq.com/help/">Datadog support</a>.
+</div>
+
+## Overview
+Parsed logs are central to be able to use Datadog Log Management to its full capacity, for queries, monitors, aggregations or automatic enrichments such as sensitive data scanner.
+When you are scaling your volume of logs it can be challenging to identify and fix logs patterns that are not parsed by your pipelines.
+
+This guide walks you through three steps to be able to identify and control the volume of unparsed logs in your organization:
+
+1. [Detect an unparsed log](#detect-an-unparsed-log)
+2. [Query for unparsed logs](#query-for-unparsed-logs)
+3. [Create a metric to track for unparsed logs](#create-a-metric-to-track-fro-unparsed-logs)
+4. [Monitor the volume of unparsed logs](#monitor-the-volume-of-unparsed-logs)
+
+
+## Detect an unparsed log
+In order to detect if a specific log has been parsed by your pipelines, access the log and check for the Event Attributes panel:
```

Comment:
```suggestion
To determine if a specific log has been parsed by your pipelines, open the log and check the Event Attributes panel. If the log is unparsed, instead of showing attributes extracted from your log, the panel shows a message saying that no attributes were extracted:
```